test/python/tools/visualization/test_circuit_text_drawer.py

```diff
@@ -496,15 +501,16 @@ def test_text_barrier(self):
 
     def test_text_no_barriers(self):
         """ Drawing without plotbarriers. """
-        expected = '\n'.join(['         ┌───┐               ',
-                              'q1_0: |0>┤ H ├───────────────',
-                              '         └───┘┌───┐          ',
-                              'q1_1: |0>─────┤ H ├──────────',
-                              '              └───┘┌───┐     ',
-                              'q2_0: |0>──────────┤ H ├─────',
-                              '                   └───┘┌───┐',
-                              'q2_1: |0>───────────────┤ H ├',
-                              '                        └───┘'])
+        expected = '\n'.join(['         ┌───┐     ',
+                              'q1_0: |0>┤ H ├─────',
+                              '         ├───┤     ',
+                              'q1_1: |0>┤ H ├─────',
+                              '         ├───┤     ',
+                              'q2_0: |0>┤ H ├─────',
+                              '         └───┘┌───┐',
+                              'q2_1: |0>─────┤ H ├',
+                              '              └───┘'])
+
```

Comment:
Why the last box is not justified to the left?

Reply:
As there is a barrier over the last qubit before it's H gate.